dll/win32/shell32/folders/CDrivesFolder.cpp

```diff
@@ -297,6 +297,57 @@ HRESULT CDrivesContextMenu_CreateInstance(PCIDLIST_ABSOLUTE pidlFolder,
     return CDefFolderMenu_Create2(pidlFolder, hwnd, cidl, apidl, psf, DrivesContextMenuCallback, cKeys, hKeys, ppcm);
 }
 
+static BOOL
+getAutoRunInfo(LPCWSTR Entry, LPWSTR pszValue, DWORD cchValueLen, LPCWSTR InfFile)
+{
+    static const WCHAR s_icon[] = L"icon";
```

Comment:
If you don't use the Wine-style of declaring unicode strings (as you do below in getIconLocationForDrive()), then you can as well put directly the string in the parameters of the GetPrivateProfileStringW() call.